integration-libs/cdc/assets/translations/translations.ts

```diff
@@ -12,5 +12,5 @@ export const cdcTranslations: TranslationResources = {
 };
 
 export const cdcTranslationChunksConfig: TranslationChunksConfig = {
-  cdc: ['reconsent'],
+  cdc: ['reconsent', 'profile'],
```

Comment:
If possible, please prefix your main translation keys with `cdc`, e.g. `cdcProfile`. Too bad we have `reconsent` already without `cdc` prefix, but changing it would be a potential breaking change to customers.

Why: If you use the general term `profile`, then no other features can use this term - as you map it to only `"cdc(.json)"` lazy loadable translations chunk.

Reply:
done